### DISCUSSION TITLE
macOS 15 Sequoia

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-E7250-Hackintosh#82] [Show and tell]
Same as last year, the newest beta boots on my E7250. However, due to the lack of GPU acceleration, it is quite unusable.

![Bildschirmfoto 2024-06-15 um 06 55 52](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/assets/28839925/054d8a71-e814-449f-b804-e5b252c7170a)

**Comment by SkyrilHD** [comment]
Now with the early preview of OCLP, we finally have GPU acceleration working.

![Screenshot 2024-08-22 at 14 06 05](https://github.com/user-attachments/assets/6cea2a97-8b19-4992-a473-ab878d642022)

**Reply by reggie1852** [reply]
I appreciate all the work you have done on the E7x50 platforms.  A question.  My E7250 fans will run constantly for some of the later operating systems (Sonoma and Sequoia).  Is this something your laptops experience as well?  Any good solution (CPUFriend?), although I suspect Broadwell CPUs will struggle.

**Reply by SkyrilHD** [reply]
My guess is that newer macOS operating systems are more demanding on the CPU. At the same time, the CPU is getting older. You may want to try applying a new thermal paste. It has been 9 years since the laptop was released.

I cannot speak from personal experience, as I switched to an Apple Silicon MacBook Pro two years ago. I can barely hear the fan at all :P

**Comment by madilsharief** [comment]
Thank you for your effort. Installed in one go as usual.

**Comment by RinoCol** [comment]
Hi,

First of all, thank you for all the work you have put into this project over the years. It has been incredibly helpful in keeping the Dell E7250 usable with recent macOS versions.

I can confirm what is stated in the README: AirportItlwm was not working for me on macOS Sequoia 15.7.7. After replacing it with itlwm + HeliPort, Wi-Fi started working correctly.

However, I am still experiencing an issue with Bluetooth, which is currently not working. I am not sure whether this is related to Sequoia or my configuration. I have already enabled IntelBluetoothFirmware, BlueToolFixup, and IntelBTPatcher.

Any suggestions would be greatly appreciated.

Thanks again for maintaining and documenting this project.
